### DISCUSSION TITLE
Check by script if hotspot is active

### DISCUSSION THREAD

**Discussion by outdoorbits** [davesteele/comitup#255] [General]
Hi, 

I would like to integrate comitup deeper into little-backup-box and set a link from the web UI if the hotspot is active. Up to now I use external_callback to know that. But if I use two wifi adapters to be able to mange portal pages, this method does not work in any case.

Thank you!

**Comment by davesteele** [comment]
If I understand the situation, it should. external_callback should work regardless of the number of wifi adapters.

**Comment by outdoorbits** [comment]
Of course, I just hoped to find an easy access to that information. But I can save the last external_callback message and 
`sudo comitup -i`
to read imode, If imode=router or last external_callback=HOTSPOT, the hotspot should be up right?

**Comment by davesteele** [comment]
Ah, OK. There is a D-Bus [_get_info()_](http://davesteele.github.io/comitup/man/comitup.pdf) call in Comitup that reports connected mode. The [_comitup-cli_](http://davesteele.github.io/comitup/man/comitup-cli.pdf) interface has access to that. "_comitup-cli i_" should give you what you need in a batch mode (can't verify right now).

**Reply by davesteele** [reply]
Slight correction - it is the D-Bus _state()_ call in Comitup.
